default_warehouse_from_sale_team/i18n/es.po

```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
 msgstr "Acceso a todos los albaranes"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,create_uid:0
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_team_ids
+msgid "Choose the Journals that user with this sale team can see"
```

Comment:
- Define the Journals an user that belongs to this sales team can see
- Also translation

Reply:
Done